tests/test_density_ComplexCF.py

```diff
@@ -127,6 +127,23 @@ def test_repr(self):
         cf = freud.density.ComplexCF(1000, 40)
         self.assertEqual(str(cf), str(eval(repr(cf))))
 
+    def test_repr_png(self):
+        rmax = 10.0
+        dr = 1.0
+        num_points = 100
+        box_size = rmax*3.1
+        np.random.seed(0)
+        points = np.random.random_sample((num_points, 3)).astype(np.float32) \
+            * box_size - box_size/2
+        ang = np.random.random_sample((num_points)).astype(np.float64) \
+            * 2.0 * np.pi
+        comp = np.exp(1j*ang)
+        ocf = freud.density.ComplexCF(rmax, dr)
+        # first bin is bad
```

Comment:
Is this comment just copy-pasted? I'm actually not sure what this refers to...

Reply:
It is copy-pasted.